ChangeLog.d/arc4random_buf-implicit.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Make arc4random_buf available on NetBSD and OpenBSD when _POSIX_C_SOURCE is
+     defined. Fix contributed in #3571.
```

Comment:
Normally we don't reference the PR in the changelog - please could you remove this last sentence.

Reply:
Actually, we usually do this for external contribution. I'm not saying it's particularly useful, but that's what we've been doing so far.